js/categoryFood.js

```diff
@@ -0,0 +1,25 @@
+let catMeal=document.getElementsByClassName('categories-food')
+fetch('https://www.themealdb.com/api/json/v1/1/categories.php')
```

Comment:
You set local storage but I can't see where you use the data from it? so maybe you can check the local storage before fetch and if it is empty: fetch data from API, if not and it fully, get the data from local storage..
BTW: this way will reduce the time for your code

Reply:
done  :)